packages/components/src/Dialog/Dialog.stories.tsx

```diff
@@ -40,52 +41,48 @@ const meta = {
     dismissable: true,
     keyboardDismissable: true,
   },
-} satisfies Meta<typeof Dialog.Trigger>;
+} satisfies Meta<ModalProps>;
 
 export default meta;
 type Story = StoryObj<typeof meta>;
 
 export const Basic: Story = {
   render: args => {
     return (
-      <>
-        <Dialog.Trigger {...args}>
-          <Button variant="primary">Open</Button>
-          <Dialog closeButton>
-            <Headline>This is a headline!</Headline>
-            <Text>This is some not so very long text.</Text>
-          </Dialog>
-        </Dialog.Trigger>
-      </>
+      <Dialog.Trigger {...args}>
+        <Button variant="primary">Open</Button>
+        <Dialog closeButton>
+          <Headline slot="title">This is a headline!</Headline>
+          <Text>This is some not so very long text.</Text>
+        </Dialog>
+      </Dialog.Trigger>
     );
   },
 };
 
 export const Form: Story = {
   render: args => {
     return (
-      <>
-        <Dialog.Trigger {...args}>
-          <Button variant="primary">Open</Button>
-          <Dialog>
-            {({ close, titleProps }) => (
-              <>
-                <Headline {...titleProps}>Please log into account</Headline>
-                <Stack space={4}>
-                  <TextField label="Username" />
-                  <TextField label="Password" type="password" />
-                  <Inline space={4}>
-                    <Button variant="ghost" onPress={close}>
-                      Cancel
-                    </Button>
-                    <Button variant="primary">Login</Button>
-                  </Inline>
-                </Stack>
-              </>
-            )}
-          </Dialog>
-        </Dialog.Trigger>
-      </>
+      <Dialog.Trigger {...args}>
+        <Button variant="primary">Open</Button>
+        <Dialog>
+          {({ close }) => (
+            <>
+              <Headline>Please log into account</Headline>
```

Comment:
Is the slot missing here? Maybe we add a "Dialog.Headline" that anyways had the right slot?